roles/cluster_logging/files/kibana/provision-index.sh

```diff
@@ -4,11 +4,14 @@
 # Script to setup the default index pattern in kibana
 #
 
-set -ue
+set -e
 
-INDEX_ID="logstash-*"
+if [ ! -z "$INDEX_PATTERN" ]; then
```

Comment:
> -z STRING: the length of STRING is zero
> -n STRING: the length of STRING is nonzero

So, it's either `if [ -z "$INDEX_PATTERN" ]` or `if [ ! -n "$INDEX_PATTERN" ]`, not this one :smiley: